src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -128,6 +133,11 @@ public GenericObjectMeasureInfo()
         ///
         /// </summary>
         public object Minimum { get; set; }
+
+        /// <summary>
+        /// The Standard Deviation of property values.
+        /// </summary>
+        public double? StdDeviation { get; set; }
```

Comment:
`StdDeviation` should be spelled out which is consistently with rest of PowerShell: `StandardDeviation`